code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -551,7 +551,7 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 		my_atom.on_reagent_change()
 	return 0
 
-/datum/reagents/proc/reaction(var/atom/A, var/method=TOUCH, var/volume_modifier=0, var/remove_reagents = FALSE)
```

Comment:
Why don't you change just cryo code's remove_reagents to FALSE instead

Reply:
Wouldn't that completely disable the depletion of the mix? That's not really the intention either.

Reply:
No?